### DISCUSSION TITLE
Should invite email input consider the input valid in capital letters too?

### DISCUSSION THREAD

**Discussion by 87prashant** [ParabolInc/parabol#6994] [Feedback]
https://user-images.githubusercontent.com/106697681/182592418-38932d97-d0da-4e51-9393-3f61742cc3b0.mp4

I was writing the email name with the first capital letter many times and thinking that there is some issue in the app then I understand the situation so I think it would create an issue for some users if they do not care about the casing during typing the email.
I guess it should not consider casing here for validation.

What do others think?

**Comment by thinktrans** [comment]
Case-folding i.e. changing all the cases to lower case for processing is a standard process that many of the email field handling routines should be applying. IMO, that is the bare minimum requirement.
In addition, such handling also need to handle Internationalized Email addresses. Those have much more demanding requirements than the first capitalized letter. :) 
Happy to share more if required.
Happy coding !
